WooCommerce/Classes/Extensions/WooBasicTableViewCell+Styling.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+extension WooBasicTableViewCell {
```

Comment:
Nit: Since this is applicable only to a WooBasicTableViewCell, maybe make sense to add the extension at the end of `WooBasicTableViewCell.swift`. So we will remember these methods more easily and conform to the other cells' structure.